### PR TITLE
384 Simplify UI for copybook management

### LINKED ISSUE #384: Simplify UX for handle missing copybooks
## Description ##
As a USER, I would like to have a more consistent experience when I am dealing with missing copybooks.
Since the Cobol Language Support handles references for copybooks into the ```settings.json```
I don't need to interact with popup messages and commands because I know where I need to go to update the copybook list I'd like to use for my COBOL program.

## Acceptance Criteria ##
TBD

## Definition of Done ##
- The story is implemented according to the description.
- Unit testing done (Preferably by TDD approach, coverage should be at least 70%).
- Code review done by at least one other team member.
- Manual testing is done.
- Documentation is added and reviewed (if needed).
- The pull request is approved.
- The story is merged into the development branch.
- Tests are performed on the development branch.

## Technical details ##
This is the place for any implementation-related notes

### PR DESCRIPTION
**Pull request by zacanbrcom**
Purpose of this PR is to simplify the UI for copybook management.
More in detail:
- provide a quickfix command that redirect the user to the ```settings.json``` when a copybook cannot be resolved
- remove the UI for select zowe profile when cobol file is opened and no default profile is selected
Close #384

### CODE REVIEW

**Comment by zacanbrcom** [comment]
+ @asatklichov 
+ @grianbrcom 

NOTES: Seems that build PR with integration test failed but not sure for which reason.
Plus I saw client test failed but after 2 reruns (one on branch and another on the PR) didn't fail anymore.

**Comment by temanbrcom** [comment]
I already see that `fix: Adjust unit test with renamed profile service API #384` should start with "test", not "fix". Maybe, some other commit may be a fix

**Comment by zacanbrcom** [comment]
> https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/399#issuecomment-646543535
My p

> I already see that `fix: Adjust unit test with renamed profile service API #384` should start with "test", not "fix". Maybe, some other commit may be a fix

My plan was to apply a squash at the end but I may rebase that commit to change the type

**Comment by temanbrcom** [comment]
> > I already see that `fix: Adjust unit test with renamed profile service API #384` should start with "test", not "fix". Maybe, some other commit may be a fix
> 
> My plan was to apply a squash at the end but I may rebase that commit to change the type

It's up to you, I'll accept both options

**Comment by zacanbrcom** [comment]
> 👾

your deep knowledge of emoticons is unbelivable!  (🍕 &&🍍 = ?)

**Comment by grianbrcom** [comment]
Let me guess: :pizza: && :pineapple: = :underage:

**Comment by temanbrcom** [comment]
> Let me guess: 🍕 && 🍍 = 🔞

**Comment by temanbrcom** [comment]
> Let me guess: 🍕 && 🍍 = 🔞

I'm missing an emoji with rejecting an Italian visa for this case

**Comment by zacanbrcom** [comment]
> > Let me guess: 🍕 && 🍍 = 🔞

...aaaalmost xD (💣🔥🧯💥 = end)

**Comment by grianbrcom** [comment]
> I'm missing an emoji with rejecting an Italian visa for this case

:it: :passport_control: :no_entry_sign:

File: clients/cobol-lsp-vscode-extension/src/constants.ts [hunk 1]
```diff
@@ -34,3 +34,6 @@ export const DSN_MUSTBE_NOT_EMPTY = "Dataset name segment must not be empty.";
 export const DSN_NOMORE_8CHARS = "Dataset name segment can't be more than 8 characters.";
 export const DSN_START_PROHIBITED_CHAR = `Dataset name segment: ${SEGMENT_PLACEHOLDER} starts with a prohibited character.`;
 export const DSN_CONTAINS_PROHIBITED_CHAR = `Dataset name segment: ${SEGMENT_PLACEHOLDER} contains a prohibited character.`;
+
+// QUICK FIX CONSTANTS
```
Comment (asatklichov) [inline clients/cobol-lsp-vscode-extension/src/constants.ts:38]: const name is quite self explaining, no need comment

File: clients/cobol-lsp-vscode-extension/src/services/ProfileService.ts [hunk 2]
```diff
@@ -12,26 +12,28 @@
...
+import {SETTINGS_SECTION} from "../constants";
+import {ProfilesMap, ZoweApi} from "./ZoweApi";
 
 const DEFAULT_STATUS_TEXT = "CPY profile: undefined";
 
 export class ProfileService implements Disposable {
+
+    private static isCobolProgram(fsPath: string) {
+        const ext = path.extname(fsPath).toLocaleLowerCase();
+        return ext === ".cbl" || ext === ".cob" || ext === ".cobol";
```
Comment (asatklichov) [inline clients/cobol-lsp-vscode-extension/src/services/ProfileService.ts:28]: good candidates for reusing, can be extracted as constants

Comment (zacanbrcom) [reply clients/cobol-lsp-vscode-extension/src/services/ProfileService.ts:28]: you mean the file extensions?

Comment (asatklichov) [reply clients/cobol-lsp-vscode-extension/src/services/ProfileService.ts:28]: Yes

Comment (grianbrcom) [inline clients/cobol-lsp-vscode-extension/src/services/ProfileService.ts:28]: You could try this: `return [".cbl", ".cob", ".cobol"].includes(ext);`

Comment (zacanbrcom) [reply clients/cobol-lsp-vscode-extension/src/services/ProfileService.ts:28]: 👍 sounds good.

**Pull request merged**